.travis.yml

```diff
@@ -31,6 +31,10 @@ script:
 matrix:
   allow_failures:
    - go: tip
+   - go: "1.7"
+   - go: "1.8"
+   - go: "1.9"
+   - go: "1.10"
```

Comment:
Allow failures with Golang 1.10 or older at this moment because `go get github.com/robfig/cron@0f01e6b17` ends up import path of `import "github.com/robfig/cron/v3"`, which has trailing v3 with it. This only works with Go modules mechanism. This would be nice if simple `go get gopkg.in/robfig/cron.v3` works, but apparently, it does not.
```
> go get gopkg.in/robfig/cron.v3
go: gopkg.in/robfig/cron.v3@v3.0.0-rc1: go.mod has non-....v3 module path "github.com/robfig/cron/v3" at revision v3.0.0-rc1
go: error loading module requirements
```
A version string with trailing `-rc1` is not allowed.

When v3 of `cron` is released and the version string becomes `3.0.0` or similar ones that follow semantic versioning rule, `import "gopkg.in/robfig/cron.v3"` will eventually work and Golang 1.10 or older can successfully build.